lagtraj/forcings/conversion/targets/kpt.py

```diff
@@ -536,7 +542,7 @@ def from_era5(ds_era5, da_levels, parameters, metadata):
         "flight": metadata.case,
         "date": ds_era5["origin_datetime"].values.astype("str"),
         "source": "ERA5",
-        "source_domain": "NEEDS ADDING",
+        # "source_domain": metadata.trajectory_domain_name,
```

Comment:
Check up on need for source domain separate from trajectory metadata.

Reply:
Remove for now.

Reply:
could use `getattr(metadata, "trajectory_domain_name", "unknown-trajectory-domain")`